nucleus/payara-modules/notification-core/pom.xml

```diff
@@ -122,7 +122,7 @@
         <dependency>
             <groupId>javax.ejb</groupId>
             <artifactId>javax.ejb-api</artifactId>
-            <version>3.2</version>
+            <version>${javax.ejb-api.version}</version>
```

Comment:
Does notification core actually need this dependency as it seems a bit weird.

Reply:
Yes, ejb-api not required in notification-core module.